.github/workflows/agent-orchestrator.yml

```
@@ -0,0 +1,154 @@
+            --body "🤖 **Agent Claimed**: Starting autonomous work on this issue.
+          
+          Mode: ${{ inputs.dry_run && 'Plan-only (dry run)' || 'Full execution' }}
+          Run: [${{ github.run_id }}](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})"
+      
+      - name: Dispatch Executor
+        if: steps.preflight.outputs.skip != 'true'
+        uses: ./.github/workflows/agent-executor.yml@develop
+        with:
+          issue_number: ${{ matrix.issue.number }}
+          dry_run: ${{ inputs.dry_run || true }}
+        secrets: inherit
```

actionlint (1.7.7)

unexpected key "secrets" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)